app/src/main/java/ar/scacchipa/twittercloneapp/data/model/TweetData.kt

```diff
@@ -0,0 +1,12 @@
+package ar.scacchipa.twittercloneapp.data.model
+
+import com.google.gson.annotations.SerializedName
+
+data class TweetData(
```

Comment:
please, choose a naming format either has Wrapper or not but to have both styles is confusing

Reply:
The other Wrapper class was moved to other package.